source/js/main.js

```diff
@@ -261,29 +263,26 @@ let main = {
     }
 
     // Embed additional instances of the Join Us box that don't need an API exposed (eg: Homepage)
-    if (document.querySelectorAll(`.join-us`)) {
-      var elements = Array.from(document.querySelectorAll(`.join-us`));
-
-      if (elements.length) {
-        elements.forEach(element => {
-          var props = element.dataset;
-
-          props.apiUrl = `${networkSiteURL}/api/campaign/signups/${props.signupId ||
-            0}/`;
-
-          props.csrfToken = props.csrfToken || csrfToken;
-          props.isHidden = false;
-
-          apps.push(
-            new Promise(resolve => {
-              ReactDOM.render(
-                <JoinUs {...props} whenLoaded={() => resolve()} />,
-                element
-              );
-            })
-          );
-        });
-      }
+    var joinUsElements = document.querySelectorAll(`.join-us:not(.on-nav)`);
+    if (joinUsElements && joinUsElements.length) {
```

Comment:
`document.querySelectorAll` always returns a NodeList, so just `if (joinUsElements.length)` is enough (unlike `querySelector()`, which can return `null`). However, since we only use it to check whether to start a `forEach`, we don't even need that: `forEach` will always work, it'll just "do nothing" for an arraylike of length 0, so we don't need any conditional here, just:
```
var joinUsElements = document.querySelectorAll(`.join-us:not(.on-nav)`);
joinUsElements.forEach( element => {
  ...
});
```

Reply:
😂 i keep forgetting about this

Reply:
We don't use `joinUsElements` elsewhere so I'm gonna merge them into

```js
document.querySelectorAll(`.join-us:not(.on-nav)`).forEach(element => { ... }
```